client/js/lounge.js

```diff
@@ -448,6 +452,12 @@ $(function() {
 				.find(".unread-marker")
 				.appendTo(container);
 		}
+		if (data.msg.type === "channel_list") {
+			// Delay by 1 second so the browser has time to render all channels
+			$(target + " .chat").stop().delay(1000).animate({
+				scrollTop: 0
+			});
+		}
```

Comment:
What probably needs to be done is to disable "stick to bottom" for special channel types like this one.